task1.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+using namespace std;
+int func1(int a, int b) {
+	if (a < b) {
+		return a;
+	}
+	else return b;
```

Comment:
Else keyword is not needed. You can use ternary operator. This comment is equivalent for func2 and 3